vite.config.js

```diff
@@ -4,8 +4,4 @@ import react from "@vitejs/plugin-react-swc"
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-  test: {
-    globals: true,
-    environment: "jsdom",
-  },
```

Comment:
Add this code. We are using Vitest for testing

Reply:
Done